corgidrp/recipe_templates/l1_to_kgain.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name" : "l1_to_kgain",
+    "template" : true,
+    "drpconfig" : {
+        "track_individual_errors" : false
+    },
+    "inputs" : [],
+    "outputdir" : "",
+    "steps" : [
+        {
+            "name" : "prescan_biassub",
+            "keywords" : {
+                "return_full_frame" : true
+            }
+        },
+        {
+            "name" : "detect_cosmic_rays",
+            "calibs" : {
+                "DetectorParams" : "AUTOMATIC"
+            }
+        },
+        {
+            "name" : "correct_nonlinearity",
+            "calibs" : {
+                "NonLinearityCalibration" : "AUTOMATIC"
+            }
+        },
+		{   "name" : "calibrate_kgain",
+		    "keywords" : {
+		        "n_cal" : 3,
```

Comment:
this is going to be the default recipe for the real data, so we should use the parameters we think are best for the real data.

Reply:
I leave it to the default values

Reply:
These values will be changed to default, that means they do not appear here anymore